indico/modules/categories/templates/display/overview/base.html

```diff
@@ -0,0 +1,124 @@
+{% extends 'layout/management_page.html' %}
+
+{% block side_menu %}
+    <div class="category-overview-side-menu">
+        <form action="{{ url_for('.overview', category) }}" method="GET" id="category-overview-options">
+            <span id="calendar-container"></span>
+            <input type="hidden" id="date" name="date" value="{{ start_dt|format_date(format="YYYY-MM-dd") }}">
+            <h3>{% trans %}Display options{% endtrans %}</h3>
+            <table cellpadding="0" cellspacing="0" style="width:100%">
+                <tr>
+                    <td>{% trans %}Period:{% endtrans %}</td>
+                    <td>
+                        <select name="period">
+                            <option value="day"{% if period == 'day' %} selected{% endif %}>
+                                {% trans %}day{% endtrans %}
+                            </option>
+                            <option value="week"{% if period == 'week' %} selected{% endif %}>
+                                {% trans %}week{% endtrans %}
+                            </option>
+                            <option value="month"{% if period == 'month' %} selected{% endif %}>
+                                {% trans %}month{% endtrans %}
+                            </option>
+                        </select>
+                    </td>
+                </tr>
+                <tr>
+                    <td>{% trans %}Detail level{% endtrans %}</td>
+                    <td>
+                        <select name="detail">
+                            <option value="event"{% if detail == 'event' %} selected{% endif %}>
+                                {% trans %}agenda{% endtrans %}
+                            </option>
+                            <option value="session"{% if detail == 'session' %} selected{% endif %}>
+                                {% trans %}session{% endtrans %}
+                            </option>
+                            <option value="contribution"{% if detail == 'contribution' %} selected{% endif %}>
+                                {% trans %}contribution{% endtrans %}
+                            </option>
+                        </select>
+                    </td>
+                </tr>
+            </table>
+            <span id="applyButtonWrapper">
+                <input type="button" value="{% trans %}Apply{% endtrans %}" class="apply-options">
+            </span>
+        </form>
+        {% if subcategories %}
+            <br>
+            <h1>{% trans %}Legend{% endtrans %}</h1>
+            <div style="margin: 10px 0 30px 10px;width:180px;overflow:hidden;white-space:nowrap;">
```

Comment:
Since you said that you moved the inline CSS into CSS files, why not this one too? :)